userspace/sysdig/plugin_utils.cpp

```diff
@@ -107,6 +129,68 @@ void register_plugins(sinsp *inspector, const char *init_config)
     }
 }
 
+void load_plugin(string& name, string& init_config)
+{
+	if (name.find('/') != string::npos)
+	{
+		g_loaded_plugins[name] = make_shared<PluginLoaded>(name, init_config);
+		return;
+	}
+
+	// In case users passed "dummy" in place of "libdummy.so"
+	string soname = "lib" + name + ".so";
+
+	bool found = false;
+	for (const auto & g_plugin_dir : g_plugin_dirs)
+	{
+		if (string(g_plugin_dir.m_dir).empty())
+		{
+			continue;
+		}
+
+		tinydir_dir dir = {};
+
+		tinydir_open(&dir, g_plugin_dir.m_dir.c_str());
+
+		while (dir.has_next)
+		{
+			tinydir_file file;
+			tinydir_readfile(&dir, &file);
+
+			string fname(file.name);
+			string fpath(file.path);
+			string error;
+
+			if (fname == name || fname == soname)
+			{
+				g_loaded_plugins[name] = make_shared<PluginLoaded>(fpath, init_config);
+				found = true;
+				break;
+			}
+
+			tinydir_next(&dir);
+		}
+
+		tinydir_close(&dir);
+	}
+	if (!found)
+	{
+		throw sinsp_exception("plugin " + name + " not found. Use -Il to list all installed plugins.");
+	}
+}
+
+shared_ptr<sinsp_plugin> enable_plugin(sinsp *inspector, string& name)
+{
+	if (g_loaded_plugins.find(name) == g_loaded_plugins.end())
+	{
+		throw sinsp_exception("plugin " + name + " not loaded. Use -H to load it.");
+	}
+	auto ploaded = g_loaded_plugins[name];
+	shared_ptr<sinsp_plugin> plugin = sinsp_plugin::register_plugin(inspector, ploaded->path, ploaded->init_config.c_str());
+	g_loaded_plugins.erase(name);
```

Comment:
Why do we need to remove the plugin from the loaded ones after they've been enabled/registered?